src/code/z_kankyo.c

```diff
@@ -1,94 +1,2533 @@
+            Matrix_Scale(22.0f, 100.0f, 22.0f, MTXMODE_APPLY);
+            gDPSetPrimColor(POLY_XLU_DISP++, 0, 0, 255, 255, 255, 128);
+            gDPSetEnvColor(POLY_XLU_DISP++, 0, 255, 255, 128);
+            gSPMatrix(POLY_XLU_DISP++, Matrix_NewMtx(globalCtx->state.gfxCtx, "../z_kankyo.c", 3333),
+                      G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+            gSPSegment(POLY_XLU_DISP++, 0x08, SEGMENTED_TO_VIRTUAL(lightningTextures[sLightningBolts[i].textureIndex]));
+            func_80094C50(globalCtx->state.gfxCtx);
+            gSPMatrix(POLY_XLU_DISP++, &D_01000000, G_MTX_NOPUSH | G_MTX_MUL | G_MTX_MODELVIEW);
+            gSPDisplayList(POLY_XLU_DISP++, &D_0402CF30);
+        }
+    }
+
+    CLOSE_DISPS(globalCtx->state.gfxCtx, "../z_kankyo.c", 3353);
+}
+
+void func_800758AC(GlobalContext* globalCtx) {
+    globalCtx->envCtx.unk_E0 = 0xFF;
+
+    // both lost woods exits on the bridge from kokiri to hyrule field
+    if (((void)0, gSaveContext.entranceIndex) == 0x4DE || ((void)0, gSaveContext.entranceIndex) == 0x5E0) {
+        func_800F6FB4(4);
+    } else if (((void)0, gSaveContext.unk_140E) != 0) {
+        if (!func_80077600()) {
+            Audio_QueueSeqCmd((s32)((void)0, gSaveContext.unk_140E));
+        }
+        gSaveContext.unk_140E = 0;
+    } else if (globalCtx->soundCtx.seqIndex == 0x7F) {
+        if (globalCtx->soundCtx.nightSeqIndex == 0x13) {
+            return;
+        }
+        if (((void)0, gSaveContext.nightSeqIndex) != globalCtx->soundCtx.nightSeqIndex) {
+            func_800F6FB4(globalCtx->soundCtx.nightSeqIndex);
+        }
+    } else if (globalCtx->soundCtx.nightSeqIndex == 0x13) {
+        // "BGM Configuration"
+        osSyncPrintf("\n\n\nBGM設定game_play->sound_info.BGM=[%d] old_bgm=[%d]\n\n", globalCtx->soundCtx.seqIndex,
+                     ((void)0, gSaveContext.seqIndex));
+        if (((void)0, gSaveContext.seqIndex) != globalCtx->soundCtx.seqIndex) {
+            func_800F5550(globalCtx->soundCtx.seqIndex);
+        }
+    } else if (((void)0, gSaveContext.dayTime) > 0x4AAA && ((void)0, gSaveContext.dayTime) < 0xB71D) {
+        if (((void)0, gSaveContext.seqIndex) != globalCtx->soundCtx.seqIndex) {
+            func_800F5550(globalCtx->soundCtx.seqIndex);
+        }
+
+        globalCtx->envCtx.unk_E0 = 1;
+    } else {
+        if (((void)0, gSaveContext.nightSeqIndex) != globalCtx->soundCtx.nightSeqIndex) {
+            func_800F6FB4(globalCtx->soundCtx.nightSeqIndex);
+        }
+
+        if (((void)0, gSaveContext.dayTime) > 0xB71C && ((void)0, gSaveContext.dayTime) < 0xCAAC) {
+            globalCtx->envCtx.unk_E0 = 3;
+        } else if (((void)0, gSaveContext.dayTime) > 0xCAAC || ((void)0, gSaveContext.dayTime) < 0x4555) {
+            globalCtx->envCtx.unk_E0 = 5;
+        } else {
+            globalCtx->envCtx.unk_E0 = 7;
+        }
+    }
+
+    osSyncPrintf("\n-----------------\n", ((void)0, gSaveContext.unk_140E));
```

Comment:
```suggestion
    osSyncPrintf("\n-----------------\n");
```
If this works.

Reply:
no it does not, its an error on their part